core/arch/arm/kernel/abort.c

```diff
@@ -555,6 +560,8 @@ void abort_handler(uint32_t abort_type, struct thread_abort_regs *regs)
 
 	switch (get_fault_type(&ai)) {
 	case FAULT_TYPE_IGNORE:
```

Comment:
We should add a `FAULT_TYPE_EXTERNAL_ABORT` if we need special treatment for external aborts.

Reply:
Ok so that type of fault should be returned when encountering `CORE_MMU_FAULT_ASYNC_EXTERNAL` then?

I think we also miss the synchronous external abort cases, looking at the documentation for AARCH64: "Synchronous External abort, not on translation tablewalk or hardware update of translation table." is 0x10 in FSR.

So 0x10 and 0x12->0x17 are all external aborts. We miss these descriptions in `core/arch/arm/include/arm64.h`